products/redhat-build-of-openjdk.md

```diff
@@ -41,6 +41,13 @@ identifiers:
     - purl: pkg:redhat/java-11-openjdk-devel-debug
 
 releases:
+-   releaseCycle: "21"
+    releaseDate: 2023-11-14 # RHEL 8.9
+    eol: 2029-12-30
+    # https://access.redhat.com/downloads/content/java-21-openjdk/x86_64/package-latest
+    latest: "21.0.1.0.12-2"
+    latestReleaseDate: 2023-11-14
```

Comment:
```suggestion
    latest: "21.0.1.0.12-3"
    latestReleaseDate: 2023-11-13
```

According to Build Date in details of https://access.redhat.com/downloads/content/java-21-openjdk/21.0.1.0.12-3.el9/x86_64/fd431d51/package.